RxCocoa/iOS/UISearchBar+Rx.swift

```diff
@@ -47,6 +47,36 @@ extension UISearchBar {
         
         return ControlProperty(values: source, valueSink: bindingObserver)
     }
-}
-
+    
+    /**
+     Reactive wrapper for `searchBarSearchButtonClicked:` delegate event.
+     */
+    public var rx_searchTapped: ControlEvent<UISearchBar> {
```

Comment:
We would want to make this method consistent with the way we are wrapping delegate methods regarding naming and return value.

That means this should be:
```
public var rx_searchButtonClicked: Observable<Void>
```
... and ...
```
public var rx_cancelButtonClicked: Observable<Void>
```

Using `Void` is what you want actually, because you want to abstract it as a generic event, it shouldn't be important that it comes from a `UISearchBar`.